File: ruby/fizzbuzz.rb
# frozen_string_literal: true

(1..30).each do |v|
  m = []
  m.append('fizz') if (v % 3).zero?
  m.append('buzz') if (v % 5).zero?
  puts m.empty? ? v.to_s : m.join('')
end

File: go/fizzbuzz.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	for i := 1; i <= 30; i++ {
		m := []string{}
		if i%3 == 0 {
			m = append(m, "Fizz")
		}
		if i%5 == 0 {
			m = append(m, "Buzz")
		}

		if len(m) > 0 {
			fmt.Println(strings.Join(m, ""))
		} else {
			fmt.Println(i)
		}
	}
}
